modules/ROOT/pages/authentication-authorization/built-in-roles.adoc

```diff
@@ -1,68 +1,76 @@
-[role=enterprise-edition]
+[role=enterprise-edition aura-db-enterprise]
 [[auth-built-in-roles]]
-= Built-in roles
-:description: This page describes the roles that come pre-defined with Neo4j. 
+= Built-in roles and privileges
+:description: The default privileges of the built-in roles in Neo4j and how to recreate them if needed.
 
 //Check Mark
 :check-mark: icon:check[]
 
-Neo4j provides built-in roles with default privileges. The built-in roles and the default privileges are:
+== Introduction
+
+Neo4j provides a set of built-in roles that can be used to control access to the database.
+The built-in roles are hierarchical, with the `PUBLIC` role at the bottom and the `admin` role at the top.
```

Comment:
the `PUBLIC` role isn't really part of the hierarchy though, it definitely has privileges that isn't part of `reader` (or most of the other default roles). So it's more "we have hierarchical built-in roles and then PUBLIC slightly to the side"